src/screens/EditCommunity/EditCommunity.tsx

```diff
@@ -0,0 +1,374 @@
+            >
+              <View style={styles.avatar}>
+                <SvgXml width={20} height={20} xml={publicIcon} />
+              </View>
+
+              <View style={styles.optionDescription}>
+                <Text style={styles.itemText}>Public</Text>
+                <Text style={styles.categoryText}>
+                  Anyone can join, view, and search the posts in this community.
+                </Text>
+              </View>
+              <RadioButton
+                id="public"
+                onPress={() => setisPublic(true)}
+                value={'public'}
+                selected={isPublic}
+                color={isPublic ? theme.colors.primary : '#444'}
+                size={17}
+              />
+            </Pressable>
+
+            <Pressable
+              onPress={() => setisPublic(false)}
+              style={styles.listItem}
+            >
+              <View style={styles.avatar}>
+                <SvgXml width={24} height={24} xml={privateIcon()} />
+              </View>
+
+              <View style={styles.optionDescription}>
+                <Text style={styles.itemText}>Private</Text>
+                <Text style={styles.categoryText}>
+                  Only members invited by the moderators can join, view, and
+                  search the posts in this community.
+                </Text>
+              </View>
+              <RadioButton
+                id="private"
+                onPress={() => setisPublic(false)}
+                value={'private'}
+                selected={!isPublic}
+                color={!isPublic ? '#1054DE' : '#444'}
+                size={17}
+              />
+            </Pressable>
+          </View>
+          {!isPublic && (
+            <View style={styles.inputContainer}>
+              <View style={styles.titleRow}>
+                <Text style={styles.inputTitle}>
+                  Add members<Text style={styles.requiredField}> *</Text>
+                </Text>
+              </View>
+              <View style={styles.addUsersContainer}>
+                {selectedUserList.length > 0 && (
+                  <FlatList
+                    data={selectedUserList}
+                    renderItem={({ item }) => (
+                      <View style={styles.userItemWrap}>
+                        <View style={styles.avatarRow}>
+                          <View style={styles.avatarImageContainer}>
+                            <Image
+                              style={styles.avatarImage}
+                              source={
+                                item.avatarFileId
+                                  ? {
+                                      uri: getAvatarURL(
+                                        apiRegion,
+                                        item.avatarFileId
+                                      ),
+                                    }
+                                  : require('../../../assets/icon/Placeholder.png')
```

Comment:
is it possible to import it instead of using `require`

Reply:
Unfortunately, react native cannot use dynamic import for Image !_!.